### PR TITLE
Pop ups for portfolio

### PR DESCRIPTION
**Pull request by Rizwan-learn**
Create pop ups using HTML CSS & JS.

Pop-ups should show more in-depth details on the projects highlighted.
New Js to be created to comply with the new updates on HTML

### CODE REVIEW

File: pop-up.js [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+            <a href="${projects[cardNumber].seeLiveLink}">See Live <img src="image/tir.png" alt="See live"></i></a>
+          </button>
+          <button>
+            <a href="${projects[cardNumber].seeSourceLink}">See Source <img src="image/cartoon.png" alt="Github icon"></a>
+          </button>
+        </div>
+        </div>
+        </div>
+      </div>
+      </section>`;
```
Comment (thecodechaser) [inline pop-up.js]: - [ ] Good job working on the styles but still there is a little issue. You are supposed to stick to the [figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) as much as possible. Your design is a little different from the template. Please fix the alignment of elements, fix the design, and make sure it matches the template.

Your design

Screenshot (197)


Figma design

Screenshot (198)

File: pop-up.js [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+    document.addEventListener('click', (event) => {
+      if (event.target.id === 'close-popUp') {
+        popUp.style.display = 'none';
+      }
+    });
+  }
```
Comment (thecodechaser) [inline pop-up.js]: - [ ]  For mobile pop-up model please remove the left margin from technologies,
- [ ]  Please reduce the gap between text and icon in buttons, Make sure everything match with the Figma design.

Your design

Screenshot (202)


Figma design

Screenshot (201)

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## Hi @Rizwan-learn 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good accessibility report. ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] Please use camel case for commit messages
Bad example: "add pop-up model"
Good example: "Add pop-up model

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## Hi @Rizwan-learn 👋, 

Great work on making the changes requested by me 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: pop-up.js [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+        </div>
+        </div>
+      </div>
+      </section>`;
+  document.addEventListener('click', (event) => {
+    if (event.target.id === 'close-popUp') {
+      popUp.style.display = 'none';
+    }
+  });
+}
```
Comment (thecodechaser) [inline pop-up.js:113]: - [ ] As requested by me previously, Please fix the design and make sure it matches the template.
-  Please increase the width of your pop-up model and place some paddings
-  Please add a blur effect for the background as it is in the design.

Your design
Screenshot (204)


Figma design

Screenshot (198)

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## Hi @Rizwan-learn 👋, 

Great work on making the changes requested by me 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by keroloussamy — APPROVED** [review]
Hi @Rizwan-learn, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️
- Add all the requirements. ✔️

clap



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**